### PR TITLE
Support DataLakeStorage 11-09-18 Version.

### PR DESCRIPTION
**Pull request by junhuagu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by junhuagu** [comment]
Link to the PR from private Repo: https://github.com/Azure/azure-rest-api-specs-pr/pull/628

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
The readme.md file is missing an input section for this new swagger, as a result no content is being generated.  Please see https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/data-plane/Microsoft.StorageDataLake/readme.md#tag-package-2018-06-preview for an example.

File: specification/storage/data-plane/Microsoft.StorageDataLake/readme.go.md [hunk 1]
```diff
@@ -13,9 +13,19 @@ go:
...
   - tag: package-2018-06-preview
 ```
 
+### Tag:  package-2018-11-preview and go
+
+These settings apply only when `--tag=package-2018-11-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2018-11-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/storage/datalake/2018-11-09/$(namespace)
```
[the remark this answers is not in the document]
Comment (salameer) [reply specification/storage/data-plane/Microsoft.StorageDataLake/readme.go.md]: @junhuagu this was missed by @sarangan12 in the last review, please follow the right folder structer for this Repo as displayed in the Main Readme:

https://github.com/Azure/azure-rest-api-specs#directory-structure

[the remark this answers is not in the document]
Comment (salameer) [reply specification/storage/data-plane/Microsoft.StorageDataLake/readme.go.md]: So the folder structure is good but it seems that the Path as pointed out be @jhendrixMSFT  is wrong

[the remark this answers is not in the document]
Comment (ThomasMarquardt) [reply specification/storage/data-plane/Microsoft.StorageDataLake/readme.go.md]: This is for our GA, so there shouldn't be any references to "preview" in the paths or files.

[the remark this answers is not in the document]
Comment (jhendrixMSFT) [reply specification/storage/data-plane/Microsoft.StorageDataLake/readme.go.md]: I see, the swagger should be moved out of the preview subdirectory and into stable.  See [here](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/datalake-store/resource-manager/Microsoft.DataLakeStore) for an example.

[the remark this answers is not in the document]
Comment (ThomasMarquardt) [reply specification/storage/data-plane/Microsoft.StorageDataLake/readme.go.md]: Junhua will follow up and make the Swagger changes to remove "preview".  We will add examples in December, but need to take an exemption for now so we can publish the Swagger on Tuesday.

**Pull request merged**